benchmark/rust/README.md

```diff
@@ -0,0 +1,45 @@
+# Benchmarking Rust implementation of lisk-db library
+
+## Prerequisites
+
+To be able to successfully run benchmarks, following applications need to be installed on a system:
+
+* [Valgrind](https://valgrind.org)
+* [KCachegrind](https://kcachegrind.sourceforge.net/html/Home.html)
+
```

Comment:
I tried to run it on mac and I will share the info on `slack` channel. but I guess it would be better to add instructions for mac users.

Reply:
Done. Thank you!